lemur/certificates/cli.py

```diff
@@ -550,15 +576,18 @@ def rotate_region(endpoint_name, new_certificate_name, old_certificate_name, mes
     default=False,
     help="Persist changes.",
 )
-def reissue_command(old_certificate_name, notify, commit):
-    reissue(old_certificate_name, notify, commit)
+def reissue_command(old_certificate_name, serial_numbers, notify, commit):
```

Comment:
Thoughts on making this the last arg? In case someone is coding against the existing arg order